src/util/createStore.js

```diff
@@ -5,9 +5,19 @@
 import { applyMiddleware, createStore, compose } from 'redux';
 import getPlatformMiddleware from '../middleware/epic';
 import { fetchQueries, mergeCookies } from '../util/fetchUtils';
+import getClickTracker from './clickTracking';
 
 const noopMiddleware = store => next => action => next(action);
 
+export const clickTrackEnhancer = createStore => (reducer, initialState, enhancer) => {
+	const store = createStore(reducer, initialState, enhancer);
+	const clickTracker = getClickTracker(store);
+	document.body.addEventListener('click', clickTracker);
+	document.body.addEventListener('change', clickTracker);
```

Comment:
My initial preference is to let things go as they are currently implemented and let product teams decide whether the data is suited to their needs - I'm not sure how much we can really investigate in the abstract without product applications and user data to analyze in the near future, but feel free to add an Investigation task if you have a sense of how to approach the issue